Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -163,6 +163,7 @@ public struct AutomaticCuration {
             return nil
         }
         
+        // FIXME: The shortest path to the reference may not be applicable to the given variants traits.
```

Comment:
Do we plan on addressing this soon? Should we open a GitHub issue?

Reply:
Yes, I started working on this issue but it requires some replacement API that I introduced in a related PR so I put it on hold until those are merged. I opened [this issue](https://github.com/swiftlang/swift-docc/issues/1085) that refers to this code. I'll add a link in the code comment as well